src/components/grid/explorePhotoList/ExplorePhotoList.tsx

```diff
@@ -31,22 +34,32 @@ const ExplorePhotoList = () => {
     threshold: 0.8,
   });
 
+  useEffect(() => {
+    if (secureLocalStorage.getItem("accessToken") === null) {
+      navigate("/login");
```

Comment:
```suggestion
      const location = useLocation();
      navigate("/login", { state: {from: location.pathname}});
```
이런식으로 navgiate 시키면 현재 페이지에 대한 내용을 넘겨줄 수 있습니다. 어차피 현재 explore페이지 상관없겠지만 다른 페이지 에서 로그인후 돌아오도록 하려고 하실때 참고하시면 좋을 것 같습니다.

Reply:
확인해보고, 적용해보겠습니다. 
전달해주신 팔로잉 포스트 작업은 메인 헤더에서 누르면 보이도록 작업하겠습니다. 오늘 안으로 끝내서 pr 다시 걸어두겟습니다. 감사합니다.